public/app/percona/settings/components/Communication/Communication.service.ts

```diff
@@ -0,0 +1,8 @@
+import { EmailPayload } from '../../Settings.types';
+import { api } from 'app/percona/shared/helpers/api';
+
+export const CommunicationService = {
+  async testEmailSettings(settings: EmailPayload, email: string): Promise<void> {
+    return api.post('/v1/Settings/TestEmailAlertingSettings', { ...settings, email_to: email });
```

Comment:
Maybe I'm wrong with the repository... In grafana, we don't have an agreement to put all the endpoints in one place?

Reply:
well, not yet. We do have that in saas-ui, but not in grafana. But it's an idea for a future improvement :))